modules/op-event-pruner-reference.adoc

```diff
@@ -0,0 +1,76 @@
+// This module is included in the following assemblies:
+// * install_config/customizing-configurations-in-the-tektonconfig-cr.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="event-pruner-reference_{context}"]
+= Configuration of the event-based pruner
+
+:FeatureName: The event-based pruner
+include::snippets/technology-preview.adoc[]
+
+You can configure the pruning behavior for the event-based pruner by modifying the `tekton-pruner-default-spec` ConfigMap in the `openshift-pipelines` namespace. This ConfigMap supports both global pruning policies and optional namespace-specific overrides.
+
+The following example shows a default `tekton-pruner-default-spec` ConfigMap that uses global pruning rules:
+
+.Example of the default `tekton-pruner-default-spec` ConfigMap
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: tekton-pruner-default-spec
+  namespace: openshift-pipelines
+data:
+  global-config: |
+    enforcedConfigLevel: global
+    ttlSecondsAfterFinished: 300    <1>
+    successfulHistoryLimit: 3       <2>
+    failedHistoryLimit: 3           <3>
+    historyLimit: 5                 <4>
+# ...
+----
+<1> Delete resources 300 seconds (5 minutes) after completion.
+<2> Retain 3 successful runs.
+<3> Retain 3 failed runs.
+<4> Limits retention to 5 runs. This setting is used if status-specific limits are not defined. 
+
+To define pruning rules for individual namespaces, set `enforcedConfigLevel` to `namespace` and configure policies under the `namespaces` section. The following example applies a 60 second TTL to resources in the `dev-project` namespace:
+
+.Example of `tekton-pruner-default-spec` ConfigMap 
+[source,yaml]
+----
+# ...
+data:
+  global-config: |
+    enforcedConfigLevel: namespace
+    ttlSecondsAfterFinished: 300
+    namespaces:
+      dev-project:
+        ttlSecondsAfterFinished: 60
+# ...
+----
+
+You can use the following parameters in the `global-config` section of the `tekton-pruner-default-spec` ConfigMap:
+
+[cols="1,3",options="header"]
+|===
+|Parameter |Description
+
+|`ttlSecondsAfterFinished`
+|Delete resources a fixed number of seconds after they complete.
+
+|`successfulHistoryLimit`
+|Retain the specified number of the most recent successful runs. Delete older successful runs.
+
+|`failedHistoryLimit`
+|Retain the specified number of the most recent failed runs. Delete older failed runs.
+
+|`historyLimit`
+|Apply a generic history limit when status-specific limits are not defined.
+
+|`enforcedConfigLevel`
```

Comment:
|`enforcedConfigLevel`
|Specifies the level at which the pruner applies the configuration. Accepted values: `global` or `namespace`.